linux_os/guide/system/software/integrity/crypto/configure_openssl_crypto_policy/oval/shared.xml

```diff
@@ -0,0 +1,29 @@
+<def-group>
+  <definition class="compliance" id="configure_openssl_crypto_policy" version="1">
+    <metadata>
+      <title>Configure OpenSSL to use System Crypto Policy.</title>
```

Comment:
@matejak please remove `.` from the titlle

Reply:
done.